src/game/server/instagib/sql_stats_player.h

```diff
@@ -61,6 +61,12 @@ class CSqlStatsPlayer
 	int m_TicksCaught; // TODO: this is not tracked yet
 	int m_TicksInGame; // TODO: this is not tracked yet
 
+	/*************************************
+	 * foot                              *
+	 *************************************/
+
+	int m_WrongGoal;
```

Comment:
Go plural here like all the other stats

```suggestion
	int m_WrongGoals;
```